spec/requests/partners/profiles_requests_county_spec.rb

```diff
@@ -0,0 +1,112 @@
+require "rails_helper"
+
+RSpec.describe "/partners/profiles", type: :request do
+  describe "basic" do
+    let(:organization) { create(:organization, name: "Favourite Bank", partner_form_fields: []) }
+
+    let(:partner) { create(:partner, name: "Partnerrific", organization: organization) }
+    let(:partner_user) { partner.primary_user }
+    before do
+      sign_in(partner_user)
+    end
+
+    describe "on show" do
+      it "displays the partner area served entry if there are no partials specified for the organization" do
+        get partners_profile_path(partner)
+        expect(response.body).to include("Area Served")
+      end
+
+      it "handles empty county list" do
+        get partners_profile_path(partner)
+        expect(response.body).to include("No County Specified")
+      end
+    end
+
+    describe "GET #edit" do
+      it "displays the partner area served entry if there are no partials specified for the organization" do
+        get edit_partners_profile_path(partner)
+        expect(response.body).to include("Area Served")
+      end
+    end
+  end
+
+  describe "partial (area served) absence when only other partials specified" do
+    let(:organization) { create(:organization, name: "Bank of Favourville", partner_form_fields: ["media_information"]) }
+    let(:partner) { create(:partner, name: "Partnertonic", organization: organization) }
+    let(:partner_user) { partner.primary_user }
+    before do
+      sign_in(partner_user)
+    end
+    describe "on show" do
+      it "does not display the client share if only other partials are specified" do
+        partner.organization = organization
+        get partners_profile_path(partner)
+        expect(response.body).to_not include("Area Served")
+      end
+    end
+
+    describe "on edit" do
+      it "does not display the client share if only other partials are specified" do
+        partner.organization = organization
+        get edit_partners_profile_path(partner)
+        expect(response.body).to_not include("Area Served")
+      end
+    end
+  end
+
+  describe "partial (area_served) presence when that partial specified" do
+    let(:organization) { create(:organization, name: "3rd National Bank of Favour", partner_form_fields: ["area_served"]) }
+    let!(:partner) { create(:partner, name: "Partnerlicious", organization: organization) }
+    let(:partner_user) { partner.primary_user }
+    before do
+      sign_in(partner_user)
+    end
+    describe "on show" do
```

Comment:
Can we add some spacing above the `describe` blocks? It's a bit hard to read right now.

Reply:
Yup.  Next push